src/frontend.js

```diff
@@ -5,7 +5,7 @@ const next = require("next");
 const dev = process.env.NODE_ENV !== "production" && !process.env.NOW;
 
 function createNextApp() {
-  const app = next({ dev, dir: __dirname });
+  const app = next({ dev, dir: `${__dirname}/../` });
```

Comment:
Will this work in Windows? Maybe we should use `path.join` here for safety.

Reply:
It's a good point. I'm not sure. I'll look into changing it to `path.join`.

Reply:
Done.